src/components/comments/CommentForm.js

```diff
@@ -15,17 +15,16 @@ export const CommentForm = (props) => {
     }
 
     const constructNewComment = () => {
-        const post_id = parseInt(props.match.params.postId)
+        const post_id = props.postId
         if(comment.subject && comment.content){
             const newCommentObject = {
                 subject: comment.subject,
                 content: comment.content,
                 post_id,
                 user_id: parseInt(localStorage.getItem("rare_user_id")),
-                timestamp: Date.now()
             }
             addComment(newCommentObject)
-                .then(props.history.push(`/comments/${post_id}`))
+                .then(props.getCommentsForPost)
         }else{
```

Comment:
I think here you just need to add
```
.then(()=> {
                const newComment = {}
                setComment(newComment)
                document.getElementById("commentForm").reset()
            })

Reply:
This clears out the subject but not the content for me--am I dropping it in there incorrectly?